srcpkgs/arch-install-scripts/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'arch-install-scripts'
```

Comment:
I'm fine with accepting `pacman`, but I don't like accepting those install scripts.

I don't have strong opinion, anyway.

Reply:
I just added it since I thought that my use case wouldn't be accepted, if you and the other maintainers would rather it removed, I don't mind.

Reply:
`apt` and `portage` are accepted.

Reply:
Ok, I removed them.

Reply:
FWIW: The arch-install-scripts are basically like debootstrap. Their only use is for installing chroots, in contrast to pacman which *could* be misused to scribble on xbps-tracked `/usr`.

Reply:
@eli-schwartz: I guess my acceptance to this PR is not that broken?
Please correct me if I were wrong. I'm not accepting `arch-install-scripts` is correct action, I guess?

Reply:
The pacman package is useful either way. For example, you can manage *existing* archlinux chroots with `/usr/bin/pacman --sysroot /path/to/chroot`.

However, I would argue that accepting `arch-install-scripts` is the correct action, and would encourage adding it in a followup PR. This would make it possible to create *new* archlinux chroots with `/usr/bin/pacstrap /path/to/chroot`.

Reply:
I'm sorry in advance if this sounds arguative.
I think `arch-install-scripts` could be downloaded to some unnamed directories.
From there, we can bootstrap a new ArchLinux chroot.

I've never used `arch-install-scripts` but I've used `debootstrap` (for some definition of use) and I think it's not necessary to be installed system-wide.

Based on that, I think I wasn't wrong to discourage `arch-install-scripts`

Reply:
That's true, pacstrap is a shellscript which is fairly trivial to download and build with m4 and make. You can run it directly from the git checkout. Minimal fuss.

There is still convenience value in having it installable, but that needs to be balanced against "yet-another-package" and "who's-going-to-review/maintain-this". I can hear both sides. :)